bin/plugin/commands/packages.js

```diff
@@ -449,17 +454,16 @@ async function runPackagesRelease( config, customMessages ) {
 		...customMessages
 	);
 
-	const temporaryFolders = [];
 	if ( config.interactive ) {
 		await askForConfirmation( 'Ready to go?' );
-
-		// Cloning the Git repository.
-		config.gitWorkingDirectoryPath = await runGitRepositoryCloneStep(
-			config.abortMessage
-		);
-		temporaryFolders.push( config.gitWorkingDirectoryPath );
 	}
 
+	// Cloning the Git repository.
+	config.gitWorkingDirectoryPath = await runGitRepositoryCloneStep(
```

Comment:
We could probably avoid using repository clone on CI, but I'm not 100% sure it would be bulletproof so let's skip it for now.